libbpf-tools/memleak.c

```diff
@@ -0,0 +1,707 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+// Copyright (c) 2022 Jackie Dinh
+//
+// Based on memleak(8) from BCC by Sasha Goldshtein
+#include <argp.h>
+#include <errno.h>
+#include <signal.h>
+#include <stdio.h>
+#include <sys/resource.h>
+#include <time.h>
+#include <unistd.h>
+
+#include <bpf/libbpf.h>
+#include <bpf/bpf.h>
+
+#include "memleak.h"
+#include "memleak.skel.h"
+#include "trace_helpers.h"
+#include "uprobe_helpers.h"
+
+#define PATH_MAX	4096
+
+static struct env {
+	bool	print_help;
+	int64_t	pid;
+	int64_t	child_pid;
+	bool	trace_all;
+	int	interval;
+	int	count;
+	bool	show_allocs;
+	uint64_t	min_age_ns;
+	char	*command;
+	bool	combined_only;
+	bool	wa_missing_free;
+	int	sample_rate;
+	int	top;
+	int	min_size;
+	int	max_size;
+	char	*obj;
+	bool	percpu;
+	bool	kernel_trace;
+	int	max_stack_depth;
+} env = {
+	.print_help = false,
+	.pid = 0,
+	.child_pid = 0,
```

Comment:
You just need to initialize non-zero values explicitly.

Reply:
Fix in commit [2beb289e](https://github.com/iovisor/bcc/pull/3828/commits/2beb289e98d651bddc2127299c5627b5d6149fce).